src/core/dbus-manager.c

```diff
@@ -2390,6 +2435,7 @@ const sd_bus_vtable bus_manager_vtable[] = {
         SD_BUS_METHOD("SetExitCode", "y", NULL, method_set_exit_code, SD_BUS_VTABLE_UNPRIVILEGED),
         SD_BUS_METHOD("LookupDynamicUserByName", "s", "u", method_lookup_dynamic_user_by_name, SD_BUS_VTABLE_UNPRIVILEGED),
         SD_BUS_METHOD("LookupDynamicUserByUID", "u", "s", method_lookup_dynamic_user_by_uid, SD_BUS_VTABLE_UNPRIVILEGED),
+        SD_BUS_METHOD("GetUnitLinks", "sb", "as", method_get_unit_links, SD_BUS_VTABLE_UNPRIVILEGED),
```

Comment:
not that the order matters much, but I'd probably move this up, to the block of functions dealing with unit file installation. i.e. right before the line for SetExitCode() or so...

Oh, and humm, this should probably be called GetUnitFileLinks(), not just GetUnitLinks(). Sorry for the confusion.